src/FACT_internal.c

```diff
@@ -2817,7 +2817,8 @@ uint32_t FACT_INTERNAL_ParseSoundBank(
 	cur = 0;
 
 	/* Simple Cue data */
-	FAudio_assert(cueSimpleCount == 0 || (ptr - start) == cueSimpleOffset);
+	if (cueSimpleCount > 0)
+		ptr = start + cueSimpleOffset;
```

Comment:
We could probably indent the whole block here, even though it adds to the diff size